src/components/button/Styles.scss

```diff
@@ -1,9 +1,14 @@
 @import '../../styles/utils/colors';
 
-.submitButton{
+.submitButton {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
   text-align: center;
+  justify-content: center;
+  align-items: center;
   color: white;
-  line-height: 50px;
+  line-height: 40px;
+  max-width: 100px;
```

Comment:
What if the button text is longer like a promitonal button that sauys 'Sign me up!' for example? Would this now overflow the button?